### PR TITLE
Add skip push flag into send message/reaction

### PR DESCRIPTION
**Pull request by ferhatelmas**
If flag is true, push notification is not sent.

### CODE REVIEW

File: src/channel.ts [hunk 1]
```diff
@@ -414,9 +414,12 @@ export class Channel<
     if (!reaction || Object.keys(reaction).length === 0) {
       throw Error(`Reaction object is missing`);
     }
+
+    const { skip_push, ...rest } = reaction;
```
Comment (mahboubii) [inline src/channel.ts:418]: a bit weird to put this into the reaction object, maybe we should change the function signature to `sendReaction(id, reaction, options)`? we have a major release coming up so we can unify this across all similar function making them future proof

Comment (ferhatelmas) [reply src/channel.ts:418]: nice, if we can change signature, I would prefer it too.

Comment (mahboubii) [reply src/channel.ts:418]: what do you think @vishalnarkhede?

Comment (ferhatelmas) [reply src/channel.ts:418]: will do this change and will also introduce options to `sendMessage` for the same reason (not to mix control keys with custom fields). Sounds good ?

Comment (mahboubii) [reply src/channel.ts:418]: sounds good

Comment (ferhatelmas) [reply src/channel.ts]: updated

**Review by mahboubii — APPROVED** [review]
🌮

**Pull request merged**